microsoft-edge/extensions-chromium/getting-started/part1-simple-extension.md

```diff
@@ -172,30 +140,18 @@ Finally, register the pop-up in `manifest.json` under `browser_action` (in Manif
 }
 ```
 
-#### [Manifest V2](#tab/v2)
 
-```json
-{
-    "name": "NASA picture of the day viewer",
-    "version": "0.0.0.1",
-    "manifest_version": 2,
-    "description": "An extension to display the NASA picture of the day.",
-    "icons": {
-        "16": "icons/nasapod16x16.png",
-        "32": "icons/nasapod32x32.png",
-        "48": "icons/nasapod48x48.png",
-        "128": "icons/nasapod128x128.png"
-    },
-    "browser_action": {
-        "default_popup": "popup/popup.html"
-    }
-}
-```
+<!-- ====================================================================== -->
+## Next steps
 
----
+That's everything you need to develop a working extension.  Now, continue on to sideload and test your extension.  For more information, see [Sideload an extension](extension-sideloading.md).
```

Comment:
link to Part 2 of tut?
rendered section:
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/getting-started/part1-simple-extension?branch=pr-en-us-3277#next-steps
@bhuvanapriyap

Reply:
Ans: the next step is not tut Part 2.  don't need link in Next Step section